docs/manual/other-topics/legacy.md

```diff
@@ -52,7 +52,7 @@ Collection.init({
 }, { sequelize });
 ```
 
-And if your model has no primary key at all you can use `Model.removeAttribute('id');`
+And if your model has no primary key at all you can use `Model.removeAttribute('id');`.  Instances without primary keys, retrieved using `findOne()`, store the exact query criteria in the model class: once many instances are queried simultaneously, doing instance.save() on any of the instances, will always update the last retrieved instance.  The mitigation is to use `instance.update()` and provide there the search criteria.  [#13819](https://github.com/sequelize/sequelize/pull/13819) proposes a fix, storing the query per instance.
```

Comment:
I'd prefer to include a disclaimer instead, as the ability to call these instance methods on models that don't have a primary keys will be removed in a future release.

```suggestion
And if your model has no primary key at all you can use `Model.removeAttribute('id');`.

Instances without primary keys can still be retrieved using `Model.findOne` and `Model.findAll`.\
While it's currently possible to use their instance methods (`instance.save`, `instance.update`, etc…), doing this is discouraged, deprecated, and will lead to subtle bugs.

If your model has no primary keys, you need to use the static equivalent of the following instance methods, and provide your own `where` parameter:
- `instance.save`: `Model.update`
- `instance.update`: `Model.update`
- `instance.reload`: `Model.findOne`
- `instance.destroy`: `Model.destroy`
- `instance.restore`: `Model.restore`
- `instance.decrement`: `Model.decrement`
- `instance.increment`: `Model.increment`
```

Reply:
You can adjust the text as you want. The proposed changes are a matter of personal style.  I do not see a need to make further changes myself.

Reply:
I'll close this PR in favor of https://github.com/sequelize/website/pull/5 then